tests/unit/tags.t

```diff
@@ -622,6 +622,41 @@ is_deeply(
 	}
 ) or diag explain $tag_ref;
 
+# test remove_tags_from_fields
+
+# removing 2 tags in a list of 4
+$product_ref = {
+	lc => "fr",
+	categories => "pommes, bananes, en:pears, fr:fraises, es:limones",
+};
+
+remove_tag_from_field($product_ref, "fr", "categories", "bananes, fraises");
+is_deeply($product_ref->{categories}, "pommes, oranges");
```

Comment:
This is a scalar, so you should use **is**

Reply:
Why would you expect "pommes, oranges" here ? I would expect "pommes, en:pears, es:limones")

Reply:
Yes, it should! It was a mistake.